### PR TITLE
Added networth to api.

### LINKED ISSUE #182: apis should return the net worth of users
api calls to `/api/investor/nameoftheuser`  return only the user balance, and not the net worth.

### PR DESCRIPTION
**Pull request by thecsw**
I hope it's right!
Closes #182

### CODE REVIEW

File: src/api.py [hunk 1]
```diff
@@ -161,23 +161,31 @@ def investors_top():
...
+    filter(Investor.name == name).\
+    first()
 
     if not sql:
         return not_found("User not found")
 
     res = {
         "name": sql.name,
         "balance": sql.balance,
+        "networth": int(x.networth),
```
Comment (rickles42) [inline src/api.py]: Should be `sql.networth`

**Review by rickles42 — CHANGES_REQUESTED** [review]

**Review by rickles42 — APPROVED** [review]

**Comment by rickles42** [comment]
There's still a problem - it throws an exception now if you try to request a user that doesn't exist, instead of returning a 404 error like it did before the change. Build and run the bot locally with your change and see if you can figure out a fix

**Comment by thecsw** [comment]
Okay!

**Comment by thecsw** [comment]
But shouldn't SQLAlchemy return `False` on error?

``` python
     if not sql:
         return not_found("User not found"
```

**Comment by rickles42** [comment]
That is indeed the behavior we want, but it's not occurring. If you test the change locally you'll see that it throws an exception instead.

**Pull request closed without merging**